apps/blog/README.md

```
@@ -1,83 +1,107 @@
+```markdown
+---
+title: "Post Title"
+date: "2025-01-15"
+category: "Data Engineering"
+tags: ["rust", "clickhouse"]
+series: "series-name"
+---
+```
 
-### `4` Configuring Auth0
+3. Write content in Markdown or MDX
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/2/3

(MD029, ol-prefix)